pkg/operator/controller/status_test.go

```diff
@@ -347,3 +405,142 @@ func TestOperatorStatusesEqual(t *testing.T) {
 		}
 	}
 }
+
+func TestComputeOperatorStatusVersions(t *testing.T) {
+	type versions struct {
+		operator, coreDNSOperand, openshiftCLIOperand string
+	}
+
+	testCases := []struct {
+		description       string
+		oldVersions       versions
+		curVersions       versions
+		allDNSesAvailable bool
+		expectedVersions  versions
+	}{
+		{
+			description:       "initialize versions, operator is avaiable",
```

Comment:
Here and below, it should be "DNSes available" (or "DNSes not all available", as appropriate).  Saying "operator is available" could be confusing because the operator's `Available` condition will be true if _any_ DNS is available, but `versions` should be updated only if _all_ of the DNSes are available.  (The actual test inputs and expectations look correct; I'm commenting only on the descriptions.)